### ISSUE TITLE
Assure that the bounds are given as a list

### ISSUE THREAD

**Issue by DSamuylov** [CMA-ES/pycma#24]
If bounds are defined as follows:

```
cma.fmin(
        cost, init, sigma,
        args=args,
        options={
            "bounds": [l_bounds, u_bounds]
        })
```

 where `l_bounds` and `u_bounds` are numpy error,  the following error is thrown:

> ~/Documents/code/python/pycma/cma/constraints_handler.py in has_bounds(self)
>     106         """return `True` if any variable is bounded"""
>     107         bounds = self.bounds
> --> 108         if bounds in (None, [None, None]):
>     109             return False
>     110         for ib, bound in enumerate(bounds):
> 
> ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()

Could you raise an appropriate error for easier debugging? Maybe add conversion into a list at the beginning of `fmin` function or in `constraints_handler`?

**Comment by nikohansen** [comment]
I believe this issue should be solved with the current version, including the one available via `pip`. Can you check your `cma.__version__` or `pip update cma`? Otherwise, can you provide the code how `l_bounds` and `u_bounds` were assigned precisely.

**Comment by DSamuylov** [comment]
I just cloned the repository and installed it from the master branch:

> (py36) denis:pycma $ python setup.py develop
> running develop
> running egg_info
> creating cma.egg-info
> writing cma.egg-info/PKG-INFO
> writing dependency_links to cma.egg-info/dependency_links.txt
> writing top-level names to cma.egg-info/top_level.txt
> writing manifest file 'cma.egg-info/SOURCES.txt'
> reading manifest file 'cma.egg-info/SOURCES.txt'
> writing manifest file 'cma.egg-info/SOURCES.txt'
> running build_ext
> Creating /Users/denis/anaconda2/envs/py36/lib/python3.6/site-packages/cma.egg-link (link to .)
> Adding cma 2.2.0 to easy-install.pth file
> 
> Installed /Users/denis/Documents/code/python/pycma
> Processing dependencies for cma==2.2.0
> Finished processing dependencies for cma==2.2.0

Versions:

(py36) denis@DeSaMac:~ $ python
Python 3.6.1 |Continuum Analytics, Inc.| (default, May 11 2017, 13:04:09) 
[GCC 4.2.1 Compatible Apple LLVM 6.0 (clang-600.0.57)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import cma
>>> cma.__version__
'2.2.0  $Revision: 4401 $ $Date: 2017-08-11 15:45:30 +0200 (Fri, 11 Aug 2017) $'
>>> 

Boundary are defined as follows:

```
        l_bounds = np.array([0, 0, 0, 0], dtype=np.float64)
        u_bounds = np.array([1, 1, 1, 1], dtype=np.float64)
```

**Comment by nikohansen** [comment]
As long as I pass `l_bounds`and `u_bounds` as defined above, it still works for me. 

I now see the problem though: it does not work if I combine these into a 2-D array. I think this is intentional, to avoid confusion with rows vs columns. I will implement a better error message.

**Comment by nikohansen** [comment]
Fixed with 12e6da0d6fc04152749e55816486f490cf110215 (I hope so).
